cabal.project

```diff
@@ -0,0 +1,14 @@
+index-state: 2021-03-15T00:00:00Z
+
+packages: .
+
+test-show-details: direct
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-prelude
+  tag: e200e42182778e39c2d28bc377c25696896094b6
+  --sha256: 0dg6ihgrn5mgqp95c4f11l6kh9k3y75lwfqf47hdp554w7wyvaw6
+  subdir:
+    cardano-prelude
+    cardano-prelude-test
```

Comment:
The `stack.yaml` also needs to be updated accordingly :+1:

Reply:
Updated `stack.yaml`.  Do these changes work for you?